lib/convertCircuitJsonToNormalizedAutoroutingJson.ts

```diff
@@ -175,9 +175,38 @@ export const convertCircuitJsonToNormalizedAutoroutingJson = (
     ),
   ).sort()
 
+  const netProperties: Record<number, { trace_thickness: number }> = {}
+
+  for (const el of circuitJson) {
+    if (el.type === "source_trace") {
+      const connNet = connectivityMap.getNetConnectedToId(el.source_trace_id)
+      const netNumber = connNetToNetNumber.get(connNet!)
+
+      if (!netNumber) continue
+
+      if (netProperties[netNumber]) continue
+
+      // Get trace thickness from the element or use default
+      const thickness = el.min_trace_thickness ?? 0.15
+      netProperties[netNumber] = {
+        trace_thickness: thickness,
+      }
+    }
+  }
+
+  // Ensure all nets have properties
+  for (const netNumber of netsToRoute) {
+    if (!netProperties[netNumber]) {
+      netProperties[netNumber] = {
+        trace_thickness: 0.15,
```

Comment:
Lets not insert defaults here, the caching engine is pretty unopinionated right now?